styleguide.css

```diff
@@ -11,7 +11,7 @@ html {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-smoothing: antialiased;
-    text-shadow: 1px 1px 1px rgba(0,0,255,0.004);
+    text-shadow: 0.1rem 0.1rem 0.1rem rgba(0,0,255,0.004);
```

Comment:
`rgba(0,0,255,0.004);`
missing spaces after commas.

Why `0.004` is it visible at all or should we remove it?